removed use of installation tab (#1778)
keycloak/keycloak-documentation: securing_apps/topics/oidc/java/jboss-adapter.adoc

The exception is the `credential` element.

> To make it easier for you, you can go to the {project_name} Admin Console and go to the Client details of the application this WAR is aligned with, then from _Action_ menu and select the *Download adapter config* option.

Reviewer: ```suggestion
To simplify this change, log into the Admin Console and go to the Client details of the application this WAR is aligned with. Then, from the _Action_ menu, select the *Download adapter config* option.
```
